.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: false
+
+addons:
+    apt:
+        packages:
+            - doxygen
+
+script:
+    - cd $TRAVIS_BUILD_DIR/docs && doxygen
```

Comment:
not really. but i guess yes, until fix problem about holding doxygen version and solve other review things.